plugins/digitalocean/resoto_plugin_digitalocean/collector.py

```diff
@@ -463,6 +468,31 @@ def remove_duplicates(images: List[Json]) -> List[Json]:
                 EdgeType.default: ["__tags"],
             },
         )
+
+        def get_size(droplet: Json) -> Json:
+            size = droplet["size"]
+            size["region"] = droplet["region"]["slug"]
+            return cast(Json, size)
+
+        sizes = [get_size(instance) for instance in instances]
+        sizes = remove_duplicates(sizes, "slug")
+
+        self.collect_resource(
+            sizes,
+            resource_class=DigitalOceanDropletSize,
+            attr_map={
+                "id": lambda s: s["slug"],
+                "urn": lambda s: size_id(s["slug"]),
+                "instance_type": "slug",
```

Comment:
in `id` you do `lambda s: s["slug"]` and here just `"slug"` - what's the difference?